pkg/controller/plan/util/utils.go

```diff
@@ -33,4 +40,19 @@ func CalculateSpaceWithOverhead(requestedSpace int64, volumeMode *core.Persisten
 	return spaceWithOverhead
 }
 
+func GetDeviceNumber(deviceString string) int {
+	if !strings.HasPrefix(deviceString, diskPrefix) || len(deviceString) <= len(diskPrefix) {
```

Comment:
isn't it enough to check `!strings.HasPrefix(deviceString, diskPrefix)`, why do we need the second part?

Reply:
ahh i think the `=` is redundant in the second check, it was for making sure we are in the correct format, for example if we will get only `/dev/sd` we missing the disk or partition so we will return 0.

Reply:
ok then:
```suggestion
	if !strings.HasPrefix(deviceString, diskPrefix) || len(deviceString) > len(diskPrefix) {
```